.github/CODEOWNERS

```diff
@@ -0,0 +1,21 @@
+# Lines starting with '#' are comments.
```

Comment:
cool how does this work? 
also, how maintainable is this? is using the blame feature not enough? 
My only concern with this is what happens when someone writes one file or module, then another adds another feature to it. Do they both become code owners?

Reply:
So I just stumbled across this feature and thought it would be good for things that are a little more complex or where the knowledge regarding them is centralized to one member of the team. All this does is require that if a file has a code owner, and that file changed in a PR, that code owner must be included in the review. We wouldn't necessarily have to keep adding code owners to a file as other people touch it, more likely as others get more familiar with it we could remove code owners for that file altogether :grinning:

Reply:
I'm fine with it, let's make the rest of the group aware of it later during our meeting :)